### PR TITLE
Log requests into symfony default logger

### PR DESCRIPTION
**Pull request by gregurco**
- [ ] Bug fix
- [x] New feature
- [ ] BC breaks
- [ ] Deprecations
- [x] Tests pass

Fixed tickets: [#99]
License: MIT

### CODE REVIEW

**Comment by gregurco** [comment]
@florianpreusner I tested with symfony app. It works :slightly_smiling_face:

File: README.md [hunk 1]
```diff
@@ -159,6 +159,18 @@ The service on the tag, is so that if you have multiple REST endpoints you can d
 ### Symfony Debug Toolbar / Profiler
 <img src="/src/Resources/doc/img/debug_logs.png" alt="Debug Logs" title="Symfony Debug Toolbar - Guzzle Logs" style="width: 360px" />
 
+### Symfony logs
+All request are logged into symfony default logger (`@logger` service) with next format:
```
Comment (florianpreusner) [inline README.md]: request**s**

File: README.md [hunk 2]
```diff
@@ -159,6 +159,18 @@ The service on the tag, is so that if you have multiple REST endpoints you can d
...
+```
+[datetime] eight_points_guzzle.LEVEL: {method} {uri} {code}
```
Comment (florianpreusner) [inline README.md]: maybe LEVEL can be replaced with {log_level}?

Comment (florianpreusner) [reply README.md]: also [datetime]: [{datetime}]

Comment (gregurco) [reply README.md]: good idea :+1:

**Comment by florianpreusner** [comment]
Released in v7.3.0 (https://github.com/8p/EightPointsGuzzleBundle/releases/tag/7.3.0)

**Pull request merged**